File: php/level/missing.php
<?php
function missed($thing){
    $al   = 'abcdefghijklmnopqrstuvwxyz';
    $some = strpos($al,$thing[0]);
    for ($i = 0; $i<strlen($thing);$i++){
        if($thing[$i] !== $al[$some+1]){
            return "this is it :".$al[$some+1];
        }
    }

}

echo missed('acd');

File: python/rank/missing.py
def findmissed(thing):
    al   = "abcdefghijklmnopqrstuvwxyz"
    some = al.index(thing[0])

    for i in al[some:]:
        if i not in thing:
            return "this is it => " + i
    
print(findmissed("acd"))